src/Compilers/Test/Utilities/CSharp/TestOptions.cs

```diff
@@ -32,5 +34,10 @@ public static class TestOptions
 
         public static readonly CSharpCompilationOptions UnsafeDebugDll = DebugDll.WithAllowUnsafe(true);
         public static readonly CSharpCompilationOptions UnsafeDebugExe = DebugExe.WithAllowUnsafe(true);
+
+        public static CSharpCompilationOptions WithStrictMode(this CSharpCompilationOptions options)
+        {
+            return options.WithFeatures(ImmutableArray.Create("strict"));
```

Comment:
Then the name of the method is confusing and should be changed to make it clear that it overwrites Features.